src/iroha/iroha.js

```diff
@@ -3,90 +3,207 @@
 *
 * SPDX-License-Identifier: Apache-2.0
 *
-* @file, definition of the Fabric class, which implements the caliper's NBI for hyperledger fabric
+* @file, definition of the Iroha class, which implements the Caliper's NBI for Hyperledger Iroha.
 */
 
+/* eslint-disable no-console */
```

Comment:
Now you can use log function from src/comm/util.js to resolve the eslint error instead of disable it.
We may extend this function later.